test/_validate_schema.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict'
+
+// Validate an APM server intake object against its schema.
+//
+// Usage example:
+//    const { validateSpan } = require('./_validate_schema')
+//    const errs = validateSpan(mySpanObj)
+// `errs` is null if mySpanObj is valid, else it is an array of ajv ErrorObject.
```

Comment:
REVIEW NOTE: Note that this file somewhat duplicates facilities in "test/integration/api-schema/_utils.js". However those are a little baroque, undocumented, and being async (unnecessarily), means its `fooValidator` functions are hard to use in synchronous test code.  I think this `_validate_schema.js` utility module will be a more convenient basis for doing more schema validation in our tests.